modules/storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+
+[id="reclaim-manual_{context}"]
+= Manually reclaiming a PV
+
+When a PersistentVolumeClaim (PVC) is deleted, the PersistentVolume (PV) still exists and is considered "released". But the PV is not yet available for another claim because the previous claimant's data remains on the volume.
```

Comment:
s/But/However,/ OR s/But the PV/The PV/

Is it localization-friendly to start a sentence with "But?"